.style.yapf

```diff
@@ -0,0 +1,5 @@
+# Ref: https://github.com/google/iree-llvm-sandbox/blob/main/.style.yapf
+[style]
+based_on_style = google
```

Comment:
I prefer to use the `yapf` style since it seems the preferred style for Google open source projects according to https://github.com/google/yapf#formatting-style

Also, I believe it's including `indent_width = 2` rule by default according to
https://github.com/google/yapf/blob/d017a9ebf2f973f4aad4d1a5ae83680731a60b72/yapf/yapflib/style.py#L508